spidermon/contrib/actions/sentry/sentry.py

```diff
@@ -0,0 +1,42 @@
+from spidermon.contrib.actions.sentry import SendMessage
+from spidermon.exceptions import NotConfigured
+
+from sentry_sdk.client import Client
+from sentry_sdk import configure_scope
+
+
+class SendSentryMessage(SendMessage):
```

Comment:
Is it really necessary to have two classes to implement this action (`SendSentryMessage` and `SendMessage`)? All methods in `SendMessage` looks much coupled with `SendSentryMessage`, so maybe a single class would be enough.

Reply:
Just trying to be consistent here as well. We are not sure if somebody wants to extend the behaviour in future that's why i prefered to keep it similar to existing actions.

Reply:
If they want to extend the functionality, they could just extend a unique `SendSentryMessage`. IMHO to have a separate class would make sense only if `SendMessage` is used by other actions that sends messages not related to Sentry (as an interface with `get_message` and `send_message` methods).

Reply:
Going to have single class now, since we don't need to split it by template and actions being made.